pkg/components/components.go

```diff
@@ -24,6 +25,7 @@ const (
 type AddonsImages struct {
 	Multus         string
 	LinuxBridgeCni string
+	BridgeMarker   string
```

Comment:
Keep this consistend with bridge CNI, use `LinuxBridgeMarker` instead

Reply:
done